tests/unit/records/test_base.py

```diff
@@ -22,3 +22,106 @@ def test_strip_empty_values():
     assert "empty_string" not in data_stripped
     assert "empty_array" not in data_stripped
     assert "empty_dict" not in data_stripped
+
+
+def test_split_urls():
```

Comment:
isn't it cleaner to use something like ``@pytest.mark.parametrize``

Reply:
Actually in my opinion, tests with more complex `expected values` are not good for pytest.mark.parametrize as it's making things less readable/it creates bigger mess. But this is only my opinion and if you are not agreeing with me i can change it.